### PR TITLE
Scaffold housekeep

### PR DESCRIPTION
**Pull request by Ehbraheem**
This PR adds the following changes:

- [x] Update in house packages
- [x] Update endpoint response format.
- [x] Update pagination routes.

### CODE REVIEW

File: scaffold/service/package.json.replace [hunk 1]
```diff
@@ -88,6 +87,6 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
-  "author": "{{{scaffold_project_author}}}",
+  "{{{scaffold_project_name}}}or": "{{{scaffold_project_{{{scaffold_project_name}}}or}}}",
```
Comment (jabdul) [inline scaffold/service/package.json.replace]: what?

`{{{scaffold_project_name}}}or`

File: scaffold/endpoint/{{{scaffold_entities}}}/routes.test.ts [hunk 2]
```diff
@@ -276,26 +270,25 @@ describe(`Routes: ${ROUTE_NAME}`, () => {
       mockContentType.mockImplementation(() => mockResponse);
       mockRequest = {
         log: jest.fn(),
-        params: jest.fn().mockReturnValue({
-          pageid: 1,
-        }),
         query: jest.fn().mockReturnValue({
-          name: 'name',
+          type: 'type',
```
Comment (jabdul) [inline scaffold/endpoint/{{{scaffold_entities}}}/routes.test.ts:274]: What is `type` for?

File: scaffold/service/package.json.replace (hunk 1 above)
Comment (Ehbraheem) [reply scaffold/service/package.json.replace]: Thanks. That was done by search and replace. Fixed now

File: scaffold/endpoint/{{{scaffold_entities}}}/routes.test.ts (hunk 2 above)
Comment (Ehbraheem) [reply scaffold/endpoint/{{{scaffold_entities}}}/routes.test.ts]: Actually it was supposed to be `offset`. I have made modifications now. Thanks for the observation.

**Pull request merged**